tests/internal/remoteconfig/test_remoteconfig.py

```diff
@@ -172,6 +172,41 @@ def test_remote_configuration_check_deprecated_override():
             assert str(capture[0].message).startswith("Using environment")
 
 
+@mock.patch.object(RemoteConfigClient, "_send_request")
+@mock.patch.object(RemoteConfig, "_check_remote_config_enable_in_agent")
+def test_remote_configuration_ip_blocking(mock_check_remote_config_enable_in_agent, mock_send_request):
+    class Callback:
+        features = {}
+
+        def _reload_features(self, metadata, features):
+            self.features = features
+
+    callback = Callback()
+
+    with override_env(dict(DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS="0.1")):
+        mock_check_remote_config_enable_in_agent.return_value = True
+        mock_send_request.return_value = get_mock_encoded_msg(
+            b'{"rules_data": [{"data": [{"expiration": 1662804872, "value": "127.0.0.0"}, '
+            b'{"expiration": 1662804872, "value": "52.80.198.1"}], "id": "blocking_ips", '
+            b'"type": "ip_with_expiration"}]}'
+        )
+        rc = RemoteConfig()
+        rc.register(ASM_FEATURES_PRODUCT, callback._reload_features)
+        sleep(0.15)
```

Comment:
can we manually trigger a poll?

Reply:
No idea. Will ask Alberto on monday.

Reply:
@brettlangdon If I understand correctly, we test it in Flask PR:
https://github.com/DataDog/dd-trace-py/pull/5024/files#diff-bab792ffca2b00287b885d7e3d23f474135e6f5dd8aa81002f3be06cfecadad3

Or we have unit tests in 1.x:
https://github.com/DataDog/dd-trace-py/blob/1.x/tests/internal/remoteconfig/test_remoteconfig.py#L142
https://github.com/DataDog/dd-trace-py/blob/1.x/tests/internal/remoteconfig/test_remoteconfig_client.py#L85

Do you mean that?

Reply:
I just mean, instead of setting the polling sleep interval really low, and then sleeping, can we just manually trigger a poll?

e.g. do we have `rc.poll()`/`rc._run()` or something that will make the remote config client do what we need it to without worrying about sleeping/timing stuff?

Reply:
Ok, I understand :smiley: I think it's ok to test the Remote Config worker in background. I think it's more accurate this way rather than `RemoteConfig.request()`